javalin/src/main/java/io/javalin/config/FileUploadConfig.kt

```diff
@@ -0,0 +1,87 @@
+package io.javalin.config
+
+import io.javalin.http.util.MultipartUtil
+import jakarta.servlet.MultipartConfigElement
+
+/**
+ * This class contains the configuration for handling multipart file uploads
+ *
+ * @property cacheDirectory : the directory where files which exceed the maximum in memory size should be cached
+ * @property maxFileSize : the maximum size allowed (in bytes) for an individual uploaded file
+ * @property maxInMemoryFileSize : the maximum size allowed (in bytes) before uploads are cached to disk
+ * @property maxTotalRequestSize : the maximum size allowed (in bytes) for the entire multipart request
+ */
+class FileUploadConfig {
+    //@formatter:off
```

Comment:
I don't think you have any non-standard formatting here, so you can delate these formatter comments.

Reply:
I didn't know what that did, hence why I left it - have removed it now